### ISSUE TITLE
Update the fmt progress in the same line

### ISSUE THREAD

**Issue by He-Pin** [VirtusLab/scala-cli#2372]
![image](https://github.com/VirtusLab/scala-cli/assets/501740/a85c16cd-38f2-4663-9daa-bd52df35a248)

Thanks, and i think it would be nice to have the progress updated in the same line.

**Comment by SethTisue** [comment]
Is the problem unique to scala-cli, or might it just be a standard behavior of scalafmt that occurs with scala-cli simply because it launches scalafmt for you?

**Comment by tgodzik** [comment]
Thanks for reporting! This is indeed a standard behaviour of scalafmt, this would need to be fixed upstream

**Comment by He-Pin** [comment]
@tgodzik Yes, and no, as an end user, we are using scala-cli :), and thanks.

**Comment by tgodzik** [comment]
Actually, I realised this works for me, how are you running scalafmt? Meaning, what system/terminal?

**Comment by He-Pin** [comment]
> Actually, I realised this works for me, how are you running scalafmt? Meaning, what system/terminal?

On window 11 ,run in cmd
